src/app/containers/Article/index.jsx

```diff
@@ -34,6 +34,7 @@ const metadataProps = (amp, config, id, metadata, promo, service) => {
     locale: config.locale,
     metaTags: allTags(metadata.tags),
     opengraphSiteName: config.opengraphSiteName,
+    serviceName: config.opengraphSiteName,
```

Comment:
Talking with @katikaa about the requirements, we should add `brandName` into the `serviceConfig` file and we can use that for the `<title>` as well as for the `opengraphSiteName` attribute. 
You don't need to remove the `opengraphSiteName` field for this PR.